dspace-spring-rest/src/main/java/org/dspace/app/rest/security/StatelessAuthenticationFilter.java

```diff
@@ -78,6 +78,7 @@ private Authentication getAuthentication(HttpServletRequest request) {
             if (eperson != null) {
                 //Pass the eperson ID to the request service
                 requestService.setCurrentUserId(eperson.getID());
+                context.setCurrentUser(eperson);
```

Comment:
Just a minor remarks, I don't think this is necessary as this is already done in the `EPersonClaimProvider` (https://github.com/DSpace/DSpace/blob/master/dspace-spring-rest/src/main/java/org/dspace/app/rest/security/jwt/EPersonClaimProvider.java#L46)